src/Samples/Toolkit.SampleApp.WPF_Net5/Toolkit.SampleApp.WPF_Net5.csproj

```diff
@@ -5,14 +5,23 @@
     <TargetFramework>$(NetWindowsTargetFramework)</TargetFramework>
     <UseWPF>true</UseWPF>
     <RootNamespace>Esri.ArcGISRuntime.Toolkit</RootNamespace>
+    <NoWarn>$(NoWarn);NU5105;MC3074;MSB4181;MC1000;MC3066</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
     <Content Include="**\*.png">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
-  
+
+  <ItemGroup>
+    <Compile Remove="bin\**" />
+    <Content Remove="bin\**" />
+    <EmbeddedResource Remove="bin\**" />
+    <None Remove="bin\**" />
+    <Page Remove="bin\**" />
+  </ItemGroup>
+
```

Comment:
This shouldn't be necessary. If it is there's a different problem here

Reply:
IIRC this was necessary because of the `**\*.png` above. I was seeing build errors reporting that png file couldn't be included from the bin folder; I needed to do a clean build each time to work around it.

Reply:
![image](https://user-images.githubusercontent.com/29742178/151888804-b2049e2f-64cd-43ea-a54b-32e77e94acc6.png)

Reply:
I updated this to make it simpler and the intent clearer.